### PR TITLE
Auto update CI jobs

### PR DESCRIPTION
**Pull request by rtnpro**

### CODE REVIEW

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by kbsingh** [comment]
lgtm, @bstinsonmhk can you do a quick review here and look through the process / hooks etc ?

File: ci/jjb_service_job.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+- job:
+    name: centos-container-pipeline-service-job
+    description: |
+        Managed by Jenkins Job Builder, do not edit manually!
+    node: container
+    properties:
+        - github:
+            url: https://github.com/rtnpro/container-pipeline-service
```
Comment (bstinsonmhk) [inline ci/jjb_service_job.yml]: We should probably be using https://github.com//CentOS/container-pipeline-service for this property

**Pull request merged**